src/consensus/aft/impl/execution.cpp

```diff
@@ -60,6 +62,24 @@ namespace aft
     request->serialize_message(
       NoNode, ctx->pbft_raw.data(), ctx->pbft_raw.size());
 
+    if (request_tracker != nullptr)
+    {
+      const auto& raw_request = ctx->get_serialised_request();
+      auto data_ = raw_request.data();
+      auto size_ = raw_request.size();
+
+      std::array<uint8_t, 32> hash;
+      tls::do_hash(data_, size_, hash, MBEDTLS_MD_SHA256);
```

Comment:
We have a bunch of `crypto::Sha256Hash` in other places, we ought to tidy up our hashing and have it all go through a single interface, which isn't too difficult to switch to a different digest algorithm if/when we need to.

Reply:
https://github.com/microsoft/CCF/issues/1690